src/authInfo.ts

```diff
@@ -58,6 +58,8 @@ export interface AuthFields {
   createdOrgInstance?: string;
   devHubUsername?: string;
   instanceUrl?: string;
+  instanceApiVersion?: string;
+  instanceApiVersionLastRetrieved?: number;
```

Comment:
`created` is a string (js date).  Pros:
* consistency
* human-readable in a file

Cons: extra step to parse?

Reply:
I can do that.